scrape.js

```diff
@@ -107,11 +107,24 @@ const scrapeLinkedIn = async (data) => {
           ".search-result__info .search-result__result-link",
           ".reusable-search__entity-results-list .entity-result__title-text a",
         ];
-        const profileListNodes =
-          (document.querySelectorAll(profileListSelectors[0]).length &&
-            document.querySelectorAll(profileListSelectors[0])) ||
-          (document.querySelectorAll(profileListSelectors[1]).length &&
-            document.querySelectorAll(profileListSelectors[1]));
+        let profileListNodes = undefined;
```

Comment:
`null` would be a better option here than `undefined` because it's safer to use and has a type `object` rather than `undefined`.